src/app/component/quiz/quiz.component.ts

```diff
@@ -122,6 +122,9 @@ export class Quizcomponent implements OnInit, OnDestroy {
 
   next(questionindex: number): void {
     this.carousel.next();
+    if (this.dialogService.hasModelOpen()) {
+      this.dialogService.destroy();
+    }
     const values = this.FormArray.controls[questionindex].value.radioValue;
     this.answer(questionindex, values);
     this.disabledValuesAndForm();
```

Comment:
Herer on line 132 
his.questionindex = questionindex + 1;
    if (this.questionindex + 1 == this.question.length) {
      this.submit();
    } 
    
    

Could you please check without adding 1 as i believe that it should work as increment is done before that

Reply:
yes i have removed adding 1 as it is working fine without it

Reply:
yes i have removed adding 1 as increment is done before